tests/test_helpers/test_serializers.py

```diff
@@ -46,6 +62,15 @@ def test_encode_decode_i():
     assert case == decoded
 
 
+def test_error_type_not_supported():
```

Comment:
Is having a list that contains different element types (e.g. bytes, int, float) supported? 
if not, please add it here. If so, add it in the above method.

Reply:
good point.
mixed types not supported at the moment. test and explicit check in code are added